docroot/themes/humsci/humsci_basic/src/scss/partials/base/_forms.scss

```
@@ -1,8 +1,10 @@
-label {
+label,
+.preact-label {
```

See comment above about class name.

```suggestion
.select-preact__label {
```